src/main/java/seedu/todo/logic/commands/FindCommand.java

```diff
@@ -7,21 +7,28 @@
 import seedu.todo.commons.exceptions.ValidationException;
 import seedu.todo.commons.util.StringUtil;
 import seedu.todo.logic.arguments.Argument;
+import seedu.todo.logic.arguments.FlagArgument;
 import seedu.todo.logic.arguments.Parameter;
 import seedu.todo.logic.arguments.StringArgument;
+import seedu.todo.model.tag.Tag;
 import seedu.todo.model.task.ImmutableTask;
 
+import java.util.Collection;
+import java.util.Collections;
 import java.util.List;
 import java.util.function.Predicate;
 
 public class FindCommand extends BaseCommand {
     private static final String TASK_FOUND_FORMAT = "%d %s found!";
     
     private Argument<String> keywords = new StringArgument("keywords");
+
+    private Argument<Boolean> tags = new FlagArgument("tags")
+            .flag("t");
     
     @Override
     protected Parameter[] getArguments() {
-        return new Parameter[] { keywords };
+        return new Parameter[] { keywords, tags };
     }
```

Comment:
It's quickly hacked together, need time to properly code ._. or if we are improving on this